### PR TITLE
Updated travis node versions

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ sudo: false
 language: node_js
 
 node_js:
-  - "6.9.3"
+  - node
```
Comment (phpnode) [inline .travis.yml]: Which version of node will this use by default?

**Comment by amilajack** [comment]
`node` will use the latest node version by default

File: .travis.yml (hunk 1 above)
Comment (amilajack) [inline .travis.yml]: `node` will use the latest node version by default

Comment (STRML) [inline .travis.yml]: Yeah, that's a weird artifact of `node` and `iojs` being options.
Would assume that means `7.x` today, so `6` should still be in the list.

Comment (STRML) [inline .travis.yml]: It also accepts `"stable"` which is more descriptive.

Comment (phpnode) [inline .travis.yml]: yeah i think let's just be explicit and use `4`, `6` and maybe `7`

**Comment by phpnode** [comment]
hmm node 4 seems very slow to install, i wonder if it's a lerna issue

**Comment by phpnode** [comment]
The build failed before the tests got chance to run:

```
Prepublishing packages
Errored while running BootstrapCommand.execute
Error: Command failed: /bin/sh -c npm run prepublish 
sh: 1: babel: not found
```

It also took about 10x as long as node 6 or 7. Personally if it passes on 6 and 7 I'm happy so I don't really fancy digging in to fix this. 

I do wonder if we should add the following to speed it up though:

```yaml
git:
  depth: 10
cache:
  directories:
  - node_modules
```

**Comment by amilajack** [comment]
Travis build have a reputation for not being very consistent upon config changes or new dependency changes. Maybe try running the tests later?

**Comment by phpnode** [comment]
it's still failing, let's just drop node 4 for now unless you're willing to dig into the details? It seems like possibly an issue with lerna and npm 2. i'd much rather have fast tests and there's no reason why it would work on node 6 but not 4, the problem doesn't affect consumers of the library.

**Comment by phpnode** [comment]
now 6 is failing?

**Comment by amilajack** [comment]
😭 Try rerunning the test a couple of times. As bad as it sounds, it has worked for me in the past

**Comment by phpnode** [comment]
ok, it passed when i kicked it off again but this makes me nervous :), will merge for now at least, thanks!

**Pull request merged**